libgalois/include/galois/Properties.h

```diff
@@ -182,6 +182,26 @@ ConstructPropertyViews(const std::vector<arrow::Array*>& arrays) {
       arrays, std::make_index_sequence<std::tuple_size_v<PropTuple>>());
 }
 
+namespace {
+/// This function works around a bug in NumPyBuffer (arrows wrapper around numpy
+/// arrays) by const casting the values pointer of mutable buffers.
```

Comment:
nit: can you replace "by const casting the values pointer of mutable buffers." (what) with something like "which claims a buffer is mutable but doesn't support mutable_data()." (why) or something along those lines?